main.js

```diff
@@ -43,6 +43,7 @@ module.exports = function(options) {
 		withNavigationHierarchy: false,
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
+		withoutAssetPreloading: false,
```

Comment:
good point. A lot of asset delivering functionality is now bundled inside withHandlebars.

Don't _really_ want to create a major release for this though. We could however have some logic which turns it off if we have `withAssetUrlGeneration: false`